scripts/updateSources.js

```diff
@@ -77,14 +77,6 @@ async function main() {
   const textFieldVersionSource = 'vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/GeneratedVaadinTextField.java';
   const textFieldVersion = fs.readFileSync(textFieldVersionSource,'utf-8').split(/\n/).filter(l => l.startsWith('@NpmPackage'))[0];
 
-  await visitFilesRecursive(`${mod}/${id}-integration-tests/frontend`, (source, target, content) => {
-    if (/test-template.js$/.test(target)) {
-      target = target.replace('test-template', `${id}-test-template`);
-      content = content.replace('test-template', `${id}-test-template`);
-    }
-    return [target, content];
-  });
```

Comment:
Individual `test-template` parts are removed in #4216, only individual parts are now used.